<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerWindow</class>
 <widget class="QWidget" name="ManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManagerWindow</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>851</width>
     <height>531</height>
    </rect>
   </property>
   <property name="title">
    <string>主界面</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>831</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="3">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>课程名</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_cozname</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_cozname"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>教师ID</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_teaid</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_srchTeaid">
           <property name="text">
            <string>查找</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_teaid">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>最大人数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_stumax"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>学分</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_cozCredit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>课程名称</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_update">
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>添加/修改                </string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_delete">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTableView" name="tableView_show">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_insert">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="4">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>搜课</string>
    </property>
    <property name="buddy">
     <cstring>lineEdit_search</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_search">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>293</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="placeholderText">
     <string>留空搜索全部课程</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>欢迎使用EzCourse简易公选课系统，您当前是管理员状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>0</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>201812030318 程序设计课程设计</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_search</tabstop>
  <tabstop>pushButton_search</tabstop>
  <tabstop>tableView_show</tabstop>
  <tabstop>pushButton_delete</tabstop>
  <tabstop>lineEdit_cozname</tabstop>
  <tabstop>lineEdit_teaid</tabstop>
  <tabstop>pushButton_update</tabstop>
  <tabstop>pushButton_insert</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
